src/parser/jobs/drg/modules/BloodOfTheDragon.js

```diff
@@ -213,20 +264,86 @@ export default class BloodOfTheDragon extends Module {
+			const action = getDataBy(ACTIONS, 'id', parseInt(id))
+			return <Message.Item key={idx}><Trans id={`drg.blood.delay-buff.${id}`}><ActionLink {...action} /> in {this.parser.formatDuration(window.timeToNextBuff[id])}</Trans></Message.Item>
+		})
+
+		return <Fragment>
+			{canBeDelayed && couldBeDelayed && (
+				<>
+					<Message>
+						<p><Trans id="drg.blood.delay-explain">Life of the Dragon windows should line up with your personal buffs if possible. This window could have been delayed to line up with:
+						</Trans></p>
+						<Message.List>
+							{delayBuffs}
+						</Message.List>
+					</Message>
+				</>
+			)}
+			<Table>
+				<Table.Header>
+					<Table.Row key="header">
+						<Table.HeaderCell><Trans id="drg.blood.table.time">Time</Trans></Table.HeaderCell>
+						<Table.HeaderCell><Trans id="drg.blood.table.action">Action</Trans></Table.HeaderCell>
+						<Table.HeaderCell><Trans id="drg.blood.table.statuses">Personal Buffs</Trans></Table.HeaderCell>
+					</Table.Row>
+				</Table.Header>
+				{rows}
+			</Table>
+		</Fragment>
+	}
+
 	output() {
 		if (this._lifeWindows.history.length > 0) {
+			const lotdPanels = this._lifeWindows.history.map(window => {
+				return {
+					title: {
+						key: `title-${window.start}`,
+						content: <Fragment>
+							{this.parser.formatTimestamp(window.start)} <span> - </span> <Trans id="drg.blood.windows.hits">{this.parser.formatDuration(window.duration)} long, <Plural value={window.nastronds.length} one="# Nastrond"	other="# Nastronds"	/>,	<Plural	value={window.stardivers.length} one="# Stardiver" other="# Stardivers"	/></Trans>
+						</Fragment>,
+					},
+					content: {
+						key: `content-${window.start}`,
+						content: this._windowTable(window),
+					},
+				}
+			})
+
 			return <Fragment>
-				<Trans id="drg.blood.windows.preface">
-					Each of the bullets below represents a Life of the Dragon window, indicating when it started, how long it lasted, and how many window-restricted OGCDs it contained. Ideally, each window should contain a full three <ActionLink {...ACTIONS.NASTROND}/> casts and one <ActionLink {...ACTIONS.STARDIVER}/> cast.
-				</Trans>
-				<ul>
-					{this._lifeWindows.history.map(window => <li key={window.start}>
-						<strong>{this.parser.formatTimestamp(window.start)}</strong>:&nbsp;
-						<Trans id="drg.blood.windows.hits">
-							{this.parser.formatDuration(window.duration)} long, <Plural value={window.nastronds.length} one="# Nastrond" other="# Nastronds"/>, <Plural value={window.stardivers.length} one="# Stardiver" other="# Stardivers"/>
-						</Trans>
-					</li>)}
-				</ul>
+				<Message>
+					<Trans id="drg.blood.windows.preface">
+						Each of the bullets below represents a Life of the Dragon window, indicating when it started, how long it lasted, and how many window-restricted OGCDs it contained. Ideally, each window should contain a full three <ActionLink {...ACTIONS.NASTROND}/> casts and one <ActionLink {...ACTIONS.STARDIVER}/> cast.
```

Comment:
Might be worth rewording this from "Each of the bullets below..." now that it's a table and not bullets.